packages/build/package.json

```diff
@@ -21,7 +21,7 @@
     "fs-extra": "^7.0.0",
     "glob": "^7.1.2",
     "mocha": "^5.1.1",
-    "nyc": "^13.0.1",
+    "nyc": "^13.2.0",
```

Comment:
@raymondfeng I am confused. AFAIK, this change is not needed, because `^13.0.1` will install `13.2.0` on our CI servers anyway.